### PR TITLE
Remove the default assignee in the feature_request.md

### PR DESCRIPTION
**Pull request by jiachengxu**
**What this PR does / why we need it**:
Remove the default assignee in the feature_request.md, so everytime people create issuses, they will not be assigned to @thetechnick 
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Feature request
 about: Suggest an idea for this project
 title: "[Feature] "
 labels: kind/feature
-assignees: thetechnick
+assignees: ''
```
Comment (nmiculinic) [inline .github/ISSUE_TEMPLATE/feature_request.md]: why not remove the key as well?

Comment (jiachengxu) [reply .github/ISSUE_TEMPLATE/feature_request.md]: just wanted to keep it the same as the bug_report.md

Comment (thetechnick) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I wondered already why so much is getting assigned to me :D

**Pull request merged**